content/billing/cycle.md

```diff
@@ -6,10 +6,27 @@ keywords: billing, cycle, payments, subscription
 
 You can pay for a Pro or Team subscription on a monthly or yearly billing cycle. You select your preferred billing cycle when you buy your subscription.
 
-You can't change the billing cycle from monthly to annual, or from annually to monthly.
+If you have a monthly billing cycle, you can choose to switch to an annual billing cycle. Note that it's not possible to switch from an annual billing cycle to monthly.
```

Comment:
should we note here that support could do this for them? or do we not want to publish that information?

Reply:
I can check with them to see if the request comes up a lot and add this in later if so!